### PR TITLE
ADD POB hash strategy

### PR DESCRIPTION
**Pull request by dave4506**
Gets ERC1155 balances via the graph subgraph and apply a `Math.sqrt`

One thing: 
there is a limit via the graph with `first:1000` limiting to 1000 addresses, is there a workaround?

### CODE REVIEW

File: src/strategies/pob-hash-cv/index.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { subgraphRequest } from '../../utils';
+
+export const author = 'bonustrack';
```
Comment (bonustrack) [inline src/strategies/pob-hash-cv/index.ts:3]: This should be your GitHub username
```suggestion
export const author = 'dave4506';
```

File: src/strategies/pob-hash-cv/index.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  options,
+  snapshot
+) {
+  const hashOwnersParams = {
+    hashOwners: {
+      __args: {
+        where: {
+          id_in: addresses.map(a => a.toLowerCase()),
+        },
+        first: 1000, // IS THIS ENOUGH?
```
Comment (bonustrack) [inline src/strategies/pob-hash-cv/index.ts:25]: This is enough if there is less than 1000 voters (that's often the case), you would need to paginate request to support more than 1000 voters using `first: 1000, skip: 1000`

**Comment by dave4506** [comment]
@bonustrack can I get another review? thanks!

**Review by bonustrack — APPROVED** [review]
Looks good, and fast! Can you just fix the conflict?

**Comment by dave4506** [comment]
Conflict fixed

**Pull request merged**